docs/integration_tests.md

```diff
@@ -1,119 +1,94 @@
-# Running Integration Tests
+# Integration Tests
 
-Several tests are specified in [.travis.yml](../.travis.yml). They usually follow this layout:
+## Structure of Integration Tests
 
-```yaml
-    stage: Some Integration Test (--platform=kubernetes)
-    os: linux
-    before_script:
-      # download and install kubectl and setup cluster
-    script:
-      - kubectl get nodes || travis_terminate 1
-      # finally install keptn quality gates
-      - test/test_install_on_kubernetes.sh
-      - keptn status
-      - export PROJECT=musicshop
-      - test/test_quality_gates_standalone.sh
-    after_success:
-      # clean up cluster, etc...
-      - echo "Tests were successful, cleaning up the cluster now..."
-    after_failure:
-      # print some debug info
-      - echo "Keptn Installation Log:"
-      - cat ~/.keptn/keptn-installer.log
-```
+The Integration Tests and their resources are located under `/test` directory in this repository. For running the Integration Tests, there are two main directories, which should we focus on:
+* `/test/assets` -> directory containing resources and scripts, which are used during the run of the Integration Tests
+* `/test/go-tests` -> Integration Tests and Testsuites
 
-* `before_script`: Set up `kubectl`, `keptn` CLI and create a Kubernetes cluster
-* `script`: Verify connection to the kubernetes cluster, install Keptn (e.g., `test/test_install_kubernetes_quality_gates.sh`),
-  verify connection to Keptn (`keptn status`) and run the actual test (e.g., `test/test_quality_gates_standalone.sh`)
-* `after_success`: Perform some cleanup (e.g., delete the cluster if necessary)
-* `after_failure`: Print some debug output
+Integration Tests are organized into four main Testsuites (testsuite files have `testsuite_` prefixes), where every Testsuite consists of tests which are ran on a specified machine/cluster:
```

Comment:
```suggestion
Integration Tests are organized into four main testsuites (testsuite files have `testsuite_` prefixes), where every Testsuite consists of tests that are run on a specified machine/cluster:
```
I would prefer reading `test suite` (two words instead of one) but I don't have a strong opinion on this.